spec/lib/tasks/user_spec.rb

```diff
@@ -47,4 +48,32 @@
       @rake[task_name].invoke
     end
   end
+
+  describe 'usasearch:user:update_inactive_approval_status' do
+    let(:task_name) { 'usasearch:user:update_inactive_approval_status' }
+
+    before do
+      @rake[task_name].reenable
+      User.destroy_all("email != 'inactivedeveloper@fixtures.org'")
```

Comment:
@peggles2 , you shouldn't need to destroy all the other fixtures. To ensure that specs won't fail due to `#<Emailer (class)> received :user_approval_removed with unexpected arguments`, you can add `allow` to the expectation:
```ruby
allow(Rails.logger).to receive(:info)
expect(Rails.logger).to receive(:info).with(expected_message)
```

That tells the spec, "don't fail if you receive x method with some random arguments", and only tests that the expected ones are received.

Reply:
This one, I was following what we are currently doing for update_approval_status. Lets discuss

Reply:
Doesn't seem to be needed, I took it out and the test passes beautifully and fails when I break the code.